backend/api/models/computeplan.py

```diff
@@ -97,23 +105,21 @@ def update_status(self) -> None:
             compute_plan_status = self.Status.PLAN_STATUS_FAILED
         elif self.cancelation_date or stats["canceled_count"] > 0:
             compute_plan_status = self.Status.PLAN_STATUS_CANCELED
-        elif stats["waiting_count"] == stats["task_count"]:
+        elif stats["waiting_builder_slot_count"] == stats["task_count"]:
             compute_plan_status = self.Status.PLAN_STATUS_WAITING
-        elif stats["waiting_count"] < stats["task_count"] and stats["doing_count"] == 0 and stats["done_count"] == 0:
+        elif (
+            stats["waiting_builder_slot_count"] < stats["task_count"]
```

Comment:
can't we remove this condition ? we check the `==` case in the previous elif, and it can't be `>`

Reply:
Good call!